pkg/features/list.go

```diff
@@ -28,6 +28,9 @@ var (
 	// SearchPageUI enables search page instead of search modal in UI. Frontend only.
 	SearchPageUI = registerFeature("Enable search page instead of search modal in UI", "ROX_SEARCH_PAGE_UI", false)
 
+	// PatternFlyDarkTheme enables dark theme support on PatternFly pages and components
+	PatternFlyDarkTheme = registerFeature("Enable dark theme support on PatternFly pages", "ROX_PF_DARK_THEME", false)
```

Comment:
Note that this feature flag will _only_ toggle the ability to view dark mode on PatternFly components. If the flag is disabled, users can still use dark mode on Tailwind based components like they do today.